### DISCUSSION TITLE
v2.0.1 - A few steps forward, fewer steps back

### DISCUSSION THREAD

**Discussion by 0xCUB3** [2009-Nissan-Cube/About-This-Hack#106] [Announcements]
## What's Changed
* Polish localization by @mavethee in https://github.com/0xCUB3/About-This-Hack/pull/101
* Added Sequoia image sets (dark and light) by @perez987 in https://github.com/0xCUB3/About-This-Hack/pull/102\
* Metal info is next to VRAM, space has been added by @perez987 in https://github.com/0xCUB3/About-This-Hack/pull/105
* Further optimized code and employed best practices
* Fixed extra padding bug in tooltips
* Fixed bootloaders not showing up correctly

## New Contributors
* @mavethee made their first contribution in https://github.com/0xCUB3/About-This-Hack/pull/101

**Full Changelog**: https://github.com/0xCUB3/About-This-Hack/compare/1.3.0...2.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/0xCUB3/About-This-Hack/releases/tag/2.0.1'>v2.0.1 - A few steps forward, fewer steps back</a>.</em>

**Comment by perez987** [comment]
Wow!!! Impressive cleaning of code. This app is already a very mature one. I like it more and more. Thank you!
